iron-scroll-target-behavior.d.ts

```diff
@@ -101,10 +101,10 @@ declare namespace Polymer {
     /**
      * Scrolls the content to a particular place.
      *
-     * @param left The left position
+     * @param leftOrOptions The left position or scroll options
      * @param top The top position
      */
-    scroll(left: number, top: number): void;
+    scroll(leftOrOptions: number|ScrollToOptions, top?: number): void;
```

Comment:
Is ScrollToOptions a type Closure already has?

Reply:
its a type the spec (and thus typescript) has, but im not too sure if closure knows of it or not. if it doesn't, it should :p